### PR TITLE
Fastlane: Adjust text, font and background

### PR DESCRIPTION
**Pull request by carolanitz**
(closes #173)

### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### Description

changes in the snapshothelper happened through an update so I didn't touch that file manually

### CODE REVIEW

File: fastlane/Snapfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
- scheme("VLC for iOSUITests")
+ scheme("VLC-iOS-UITests")
 
  devices([
    "iPhone 5s",
```
Comment (carolanitz) [inline fastlane/Snapfile]: we probably want to add more devices here as well 🙈

**Comment by carolanitz** [comment]
@Mikanbu only in the snapshothelper though right? I mentioned above that that was updated automatically and I can't do anything about changes in there

**Comment by Mikanbu** [comment]
@carolanitz My bad apparently I zapped your comments! Even though you didn't make the changes, do we need to commit the changes?

**Comment by carolanitz** [comment]
> @carolanitz My bad apparently I zapped your comments! Even though you didn't make the changes, do we need to commit the changes?

yes because otherwise you can't do screenshots because it tells you to update

**Comment by Mikanbu** [comment]
Urg this is quite sad but then not related to this PR.

LTGM.

**Comment by carolanitz** [comment]
merged with 8e91079a4eb056d6c4f02a223ef18efe04e547b9

**Pull request closed without merging**